docs/cncf-projects/falco/README.md

```diff
@@ -0,0 +1,2 @@
+# Falco
+
```

Comment:
```suggestion
TBC See https://github.com/falcosecurity/cncf-green-review-testing
```

I think we can add this as a follow up but we could link to the repo as a starting point.